submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionFileSourceTest.kt

```diff
@@ -219,8 +219,8 @@ class SubmissionFileSourceTest(
                 line("Type", "test")
                 line()
 
-                line("File", "folder/inner")
-                line("Type", "inner folder")
+                line("File", "folder/inner/directory")
+                line("Type", "inner directory")
```

Comment:
I think this is really specialized scenario and can be easily miss in a refactor. IMO we should have specialized test "6-3-2 submission with directories with the same name on FIRE". In that way is super clear what is that that we are testing.

Reply:
Done